code/ui/generalhud/postroundmenu/PostRoundMenu.cs

```diff
@@ -58,8 +63,21 @@ public void OpenAndSetPostRoundMenu(PostRoundStats stats)
         public void OpenPostRoundMenu()
         {
             IsShowing = true;
+            lang = ILanguage.GetActiveLanguage();
+
+            _contentLabel.Text = lang.GetTranslation("POST_ROUND_TEXT");
+
+            switch (_stats.WinningRole)
+            {
+                case "Innocents":
+                    _headerLabel.Text = lang.GetTranslation("POST_ROUND_WIN_INNOCENT");
+                    break;
+
+                case "Traitors":
+                    _headerLabel.Text = lang.GetTranslation("POST_ROUND_WIN_TRAITORS");
+                    break;
+            }
```

Comment:
Here should the team be used in the same way as suggested above

Reply:
Is it possible for other roles to win? We might want to do something similar to 
`public string GetRoleTranslation(string key)`
so that we can support different roles winning the game.

Reply:
Even if not planned on our side, it should be possible :)